static-site/content/blog/2024-12-10-auspice-1password-issue.md

```diff
@@ -7,6 +7,10 @@ sidebarTitle: "Externally-caused Auspice performance issues"
 
 Scope: https://nextstrain.org and https://auspice.us, any other custom Auspice server
 
+[January 22, 2025] We have applied a workaround for the 1Password extension
```

Comment:
Might as well link the patch, might make it easier for other folks to apply it to their code:

```suggestion
[January 22, 2025] We have applied [a workaround](https://github.com/nextstrain/auspice/pull/1932) for the 1Password extension
```

Reply:
Done:

https://github.com/nextstrain/nextstrain.org/blob/e34b0e62f33d7da4af04c990e07e1bc31619951f/static-site/content/blog/2024-12-10-auspice-1password-issue.md#L10-L11